orchestra/actions/configure.py

```diff
@@ -22,9 +22,8 @@ def _run(self, args):
             logger.warning("Previous configure probably failed, running configure script in a dirty environment")
             logger.warning(f"You might want to delete the build directory (use `orchestra clean`)")
 
-        result = run_script(self.script, quiet=args.quiet, environment=self.environment)
-        if result.returncode == 0:
-            open(self._configure_successful_path, "w").close()
+        run_user_script(self.script, environment=self.environment)
+        open(self._configure_successful_path, "w").close()
```

Comment:
Consider [Pathlib.touch](https://stackoverflow.com/questions/1158076/implement-touch-using-python) (no big deal)

Reply:
Good suggestion, I prefer it as well as it better communicates the intent